ksqldb-execution/src/main/java/io/confluent/ksql/execution/runtime/RuntimeBuildContext.java

```diff
@@ -44,7 +44,7 @@
 import org.apache.kafka.streams.StreamsBuilder;
 import org.apache.kafka.streams.kstream.Windowed;
 
-public final class KsqlQueryBuilder {
+public final class RuntimeBuildContext {
```

Comment:
nit: can we add javaodc here that echos what you wrote in the commit description (specifically with the relationship between this and `PlanBuildContext`)

Reply:
ack